pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/PulsarAdminTool.java

```diff
@@ -41,75 +38,80 @@
+    @ArgGroup(heading = "Options:%n", exclusive = false)
+    protected RootParams rootParams = new RootParams();
     protected PulsarAdminSupplier pulsarAdminSupplier;
+    private PulsarAdminPropertiesProvider pulsarAdminPropertiesProvider;
 
     @Getter
     public static class RootParams {
 
-        @Parameter(names = { "--admin-url" }, description = "Admin Service URL to which to connect.")
+        @Option(names = { "--admin-url" }, description = "Admin Service URL to which to connect.",
+                descriptionKey = "webServiceUrl")
         String serviceUrl = null;
 
-        @Parameter(names = { "--auth-plugin" }, description = "Authentication plugin class name.")
+        @Option(names = { "--auth-plugin" }, description = "Authentication plugin class name.",
+                descriptionKey = "authPlugin")
         String authPluginClassName = null;
 
-        @Parameter(names = { "--request-timeout" }, description = "Request time out in seconds for "
+        @Option(names = { "--request-timeout" }, description = "Request time out in seconds for "
                 + "the pulsar admin client for any request")
         int requestTimeout = PulsarAdminImpl.DEFAULT_REQUEST_TIMEOUT_SECONDS;
 
-        @Parameter(
-            names = { "--auth-params" },
+        @Option(names = { "--auth-params" }, descriptionKey = "authParams",
                 description = "Authentication parameters, whose format is determined by the implementation "
                         + "of method `configure` in authentication plugin class, for example \"key1:val1,key2:val2\" "
                         + "or \"{\"key1\":\"val1\",\"key2\":\"val2\"}\".")
         String authParams = null;
 
-        @Parameter(names = { "--tls-allow-insecure" }, description = "Allow TLS insecure connection")
+        @Option(names = { "--tls-allow-insecure" }, description = "Allow TLS insecure connection")
         Boolean tlsAllowInsecureConnection;
 
-        @Parameter(names = { "--tls-trust-cert-path" }, description = "Allow TLS trust cert file path")
+        @Option(names = { "--tls-trust-cert-path" }, description = "Allow TLS trust cert file path")
         String tlsTrustCertsFilePath;
 
-        @Parameter(names = { "--tls-enable-hostname-verification" },
+        @Option(names = { "--tls-enable-hostname-verification" },
                 description = "Enable TLS common name verification")
         Boolean tlsEnableHostnameVerification;
 
-        @Parameter(names = {"--tls-provider"}, description = "Set up TLS provider. "
+        @Option(names = {"--tls-provider"}, description = "Set up TLS provider. "
                 + "When TLS authentication with CACert is used, the valid value is either OPENSSL or JDK. "
                 + "When TLS authentication with KeyStore is used, available options can be SunJSSE, Conscrypt "
-                + "and so on.")
+                + "and so on.", descriptionKey = "webserviceTlsProvider")
         String tlsProvider;
-
```

Comment:
also handled by picocli?

Reply:
-v, -h is supported by picocli, and you can use it anywhere.